packages/vue3/src/components/AdvancedImage.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <img :src="url" />
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+
+export default defineComponent({
+  name: "AdvancedImage",
+  props: {
+    url: String,
```

Comment:
Is this a template for the AdvancedImage until the actual implementation is completed?

Reply:
yes